share/errmsg-utf8.txt

```diff
@@ -17967,16 +17967,16 @@ ER_IB_MSG_1241
   eng "%s"
 
 ER_IB_MSG_1242
-  eng "%s"
+  eng "Can't set redo log tablespace to be encrypted in read-only mode."
```

Comment:
Why?

Reply:
This is from 8.0.16, which added these error codes.

Reply:
Ack, but please explain in the commit message